quest-ce-qui-cest-passe.md

```diff
@@ -0,0 +1,58 @@
+# Un travail de fond ...
+
+Beaucoup de travail a tout d'abord été fait sur le code en lui même, avec assez peu de résultat visible "à l’œil nu", mais essentiel pour la suite :)
+
+En effet, comme énoncé plus haut, la ZEP-12 part du constat qu'un article et un tutoriel ont énormément de choses en commun, et qu'il n'y avait donc pas de raisons de les traiter différemment. Il était donc d'une part nécessaire de repenser les modules pour qu'ils ne fassent plus qu'un. But du jeu : plus de duplications ! Ce qui a donc été réalisé permettra dans le futur d'implémenter plus facilement des fonctionnalités, qui seront directement disponibles pour les articles et les tutoriels. Le code est même assez souple pour pouvoir accueillir d'autres types de contenus (pourquoi pas [des tribunes](https://zestedesavoir.com/forums/sujet/976/zep-13-tribune-libre/) ?). Il s'agit donc bien ici de faciliter la vie des développeurs.

```

Comment:
interface de rédaction différentes
pas de gallerie
pas de béta

rien que ces trois là forçait à des comportements totalement différent. 
Si un auteur écrit un article avant d'écrire un tuto il sera tenté de 
tout mettre dans le cadre "introduction" car il pensera que c'était 
comme pour l'article.
si un auteur écrit un article, il ne sait pas comment faire relire et 
doit donc tricher avec certaines fonctionnalités du forum (texte caché)
n'ayant pas de gallerie liée, l'auteur doit s'adapter pour ses images.

Pour le lecteur on ajoutera l'absence de génération des PDF qui vient 
s'ajouter à la longue liste des choses qui sont pas très cool dans 
l'ancienne méthode.

Le 20/09/2015 20:47, Gérard Paligot a écrit :
>
> In quest-ce-qui-cest-passe.md 
> <https://github.com/artragis/tutoriels-et-articles-les-nouveautes-de-la-version-159/pull/10#discussion_r39933023>:
>
> > @@ -0,0 +1,58 @@
> > +# Un travail de fond ...
> > +
> > +Beaucoup de travail a tout d'abord été fait sur le code en lui même, avec assez peu de résultat visible "à l’œil nu", mais essentiel pour la suite :)
> > +
> > +En effet, comme énoncé plus haut, la ZEP-12 part du constat qu'un article et un tutoriel ont énormément de choses en commun, et qu'il n'y avait donc pas de raisons de les traiter différemment. Il était donc d'une part nécessaire de repenser les modules pour qu'ils ne fassent plus qu'un. But du jeu : plus de duplications ! Ce qui a donc été réalisé permettra dans le futur d'implémenter plus facilement des fonctionnalités, qui seront directement disponibles pour les articles et les tutoriels. Le code est même assez souple pour pouvoir accueillir d'autres types de contenus (pourquoi pas [des tribunes](https://zestedesavoir.com/forums/sujet/976/zep-13-tribune-libre/) ?). Il s'agit donc bien ici de faciliter la vie des développeurs.
>
>     "la vie des développeurs" -> "et celles des auteurs qui n'ont pas
>     à s'habituer à deux modules différents"
>
> C'est plutôt les développeurs. Côté utilisateur/auteur, mise à part 
> quelques fonctionnalités, tu pouvais pas te douter qu'il n'y avait 
> vraiment rien en commun.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/artragis/tutoriels-et-articles-les-nouveautes-de-la-version-159/pull/10/files#r39933023>.
>